images/capi/ansible/roles/providers/files/etc/cloud/cloud.cfg.d/05_logging.cfg

```diff
@@ -64,4 +64,4 @@ log_cfgs:
 # this tells cloud-init to redirect its stdout and stderr to
 # 'tee -a /var/log/cloud-init-output.log' so the user can see output
 # there without needing to look on the console.
-output: {all: '| ts ''[%Y-%m-%d %H:%M:%S]'' | tee -a /var/log/cloud-init-output.log'}
+output: {all: '| python3 -c ''import sys,time;sys.stdout.write("".join(( " ".join((time.strftime("[%Y-%m-%d %H:%M:%S]", time.localtime()), line)) for line in sys.stdin )))'' | tee -a /var/log/cloud-init-output.log'}
```

Comment:
Not sure if we really want to change this back to python.
Here is the correspondent PR where it was changed: https://github.com/c445/image-builder/pull/16

> The python pipe did result in waiting for the end of the script and then
writing to the log file. Using sed writes immediately to the log file.

Reply:
this will we changed afterwards by our internal ansible role, so we don't have to adopt it in here